File: frontend/src/app/features/dashboard/applicants/dtos/applicant.response.dto.ts
import {BaseResponseDto} from '../../../../core/dtos/base-response.dto';
import {ApplicantType} from '../enums/applicant.enum';

export interface ApplicantResponseDto extends BaseResponseDto{
  firstName: string;
  lastName: string;
  personalIdNumber: string;
  emailAddress: string;
  phoneNumber: string;
  type: ApplicantType
}

File: backend/src/main/java/io/hb3nce04/keykeeperapp/feature/applicant/model/dto/ApplicantResponseDto.java
package io.hb3nce04.keykeeperapp.feature.applicant.model.dto;

import io.hb3nce04.keykeeperapp.common.model.dto.BaseDto;
import io.hb3nce04.keykeeperapp.feature.applicant.model.enums.ApplicantType;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ApplicantResponseDto extends BaseDto {
    private String firstName;

    private String lastName;

    private String personalIdNumber;

    private String emailAddress;

    private String phoneNumber;

    private ApplicantType type;
}
